pkg/wait/wait_for_ready_test.go

```diff
@@ -325,6 +322,25 @@ func peNormal(name string) ([]watch.Event, int) {
 	}, len(messages)
 }
 
+func peUnstructured(name string) ([]watch.Event, int) {
+	events, msgLen := peNormal(name)
+	for i, event := range events {
+		unC, err := runtime.DefaultUnstructuredConverter.ToUnstructured(event.Object)
+		if err != nil {
+			panic(err)
```

Comment:
Why not just fail test here with a message?

Reply:
Yes, I could. But, that requires passing down the `testing.T`. Panic in test isn't nice, because if it fires tests in that package will stop and not be reported properly.

I thought maybe it's not worth it, as this mapping to unstructured data should always pass. And to pass `testing.T` it would require additional code changes.